Fixed spelling mistakes (#197)
codatio/codat-docs: docs/underwriting/introduction.md

### ✔️ Delve into the demo app to...

- Create and submit an example loan application form,
- Create a test company via Codat’s API to represent the borrower,
- Link the test company to Codat's sandbox source of financial data,
- Fetch that data using Codat’s Enhanced Profit and Loss and Enhanced Balance Sheet endpoints,
- Use Codat’s webhooks to trigger the underwriting of a loan,
- Produce an automated underwriting decision based on custom financial data points. 

### ❌ This guide does not cover...

- Building any application form and dashboard UIs,
- Details or recommendations of various financial data points used in different underwriting models.

### About the demo app

The app's codebase is available via our [GitHub repo](https://github.com/codatio/build-guide-underwriting-be). Clone it locally to try out the application.

The codebase uses Codat and its [Assess](/assess/overview) product to make an underwriting decision. We will walk you through the setup in the Portal and the code [next](/underwriting/setting-up). 

### Why we use _Assess_

> The demo app makes use of the categorization feature of [Assess](/assess/overview). This feature analyses the full list of a company's accounts and assigns a category to each account. As such, this solves the problem of bookkeepers giving different names for the same accounting category. For example, one bookkeeper may name their marketing account Facebook Ads whereas another may call it Online Marketing. Assess will categorize a both accounts in the same way, specifically, _Expense > Operating > Marketing_ enabling lenders to use a standardized taxonomy irrespective of how the bookkeeper manages their accounts.

Reviewer: ```suggestion
The demo app makes use of the categorization feature of [Assess](/assess/overview). This feature analyses the full list of a company's accounts and assigns a category to each account. This solves the problem of bookkeepers giving different names to the same accounting category. 

For example, one bookkeeper may name their marketing account "Facebook Ads", and another may call it "Online Marketing". Assess will categorize both accounts in the same way - as _Expense > Operating > Marketing_. This enables lenders to use a standardized taxonomy, no matter how the bookkeeper manages their accounts.
```
